pennylane/qnn/keras.py

```diff
@@ -370,3 +370,15 @@ def input_arg(self):
         `Layer <https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer>`__. Set to
         ``"inputs"``."""
         return self._input_arg
+
+    def get_config(self):
+        """Returns `__init__` args used to construct this instance, allowing
+        serialization of models contianing `KerasLayer`.
+        """
+        return {
+            "qnode": self.qnode,
```

Comment:
Are we still enabling serialization if the QNode is not itself serialized (QNode serialization support should hopefully come soon!)?

Reply:
Was this comment aimed at the docstring or at the fact that the (unserialized) qnode object is being returned (l. 379)?